### DISCUSSION TITLE
Can you whitelist / opt-in to specific components?

### DISCUSSION THREAD

**Discussion by MangoMarcus** [saadeghi/daisyui#2824] [Q&A] [answered]
Eg. if I want to use the dropdown component but already have my own button component

**Comment by saadeghi** [comment, marked as answer]
Unused class names will not be included in production.

And in case you are using the same custom class name as daisyUI class names, you can use daisyUI prefix config.

Let me know if you have any questions

**Reply by MangoMarcus** [reply]
Ahh prefixes, of course! Ty
